lib/rex/parser/fs/ntfs.rb

```diff
@@ -0,0 +1,243 @@
+module Rex
+  module Parser
+
+    ###
+    #
+    # This class parses the contents of an NTFS partition file.
+    # Author : Danil Bazin <danil.bazin[at]hsc.fr> @danilbaz
+    #
+    ###
+    class NTFS
+      #
+      # Initialize the NTFS class with an already open file handler
+      #
+      def initialize(file_handler)
+        @file_handler = file_handler
+        data = @file_handler.read(4096)
+        # Boot sector reading
+        @bytes_per_sector = data[11, 2].unpack("S")[0]
```

Comment:
Please use non-native pack/unpack directives so this code is agnostic of the current machine architecture. e.g. 'v' for 16 bit LE